atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -295,7 +294,7 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
         com.google.common.base.Supplier<TimestampManagementService> managementSupplier =
                 Suppliers.compose(atlasFactory::getTimestampManagementService, timestampSupplier);
 
-        LockAndTimestampServices lockAndTimestampServices = createLockAndTimestampServices(
+        LockAndTimestampServices rawLockAndTimestampServices = createLockAndTimestampServices(
```

Comment:
nit: this isn't quite the case, as the lock and timestamp services returned here already have the bridging and lock refreshing clients in place.